akka-http2-support/build.sbt

```diff
@@ -1,13 +1,36 @@
 import akka._
+import java.nio.file.Files
+import java.nio.file.attribute.{ PosixFileAttributeView, PosixFilePermission }
 
 Dependencies.http2
+Dependencies.http2Support
 
 fork in run in Test := true
 fork in Test := true
 
 connectInput in run in Test := true
 
-javaOptions in run in Test += "-javaagent:jetty-alpn-agent-2.0.5.jar"
+javaOptions in Test += "-Dh2spec.path=" + ((target in Test).value / Dependencies.h2SpecPackageName / s"h2spec${exeIfWindows}")
```

Comment:
Passing the h2spec path via a system property was the cleanest way to avoid having any OS checks inside the TCK test.